src/main/webapp/app/app-routing.module.ts

```diff
@@ -116,6 +117,11 @@ const LAYOUT_ROUTES: Routes = [navbarRoute, ...errorRoute];
                     path: 'lti',
                     loadChildren: () => import('./lti/lti.module').then((m) => m.ArtemisLtiModule),
                 },
+                {
+                    path: 'chat',
+                    component: ExerciseChatWidgetComponent,
```

Comment:
Use a lazy import here.

Reply:
How can we do this for a component exactly? All examples seem to be for modules.

Reply:
Update: We will remove this part completely as Patrick did not want a new window but only a full size widget instead.